src/test/java/uk/co/ramp/covid/simulation/population/PopulationParametersTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.co.ramp.covid.simulation.population;
+
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import uk.co.ramp.covid.simulation.CovidParameters;
+import uk.co.ramp.covid.simulation.io.ParameterReader;
+import uk.co.ramp.covid.simulation.util.InvalidParametersException;
+import uk.co.ramp.covid.simulation.util.RNG;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertTrue;
+
+public class PopulationParametersTest {
+
+    @Test
+    public void testIsValid() throws IOException {
+        //Test that valid parameters are handled correctly
+        ParameterReader.readParametersFromFile("src/test/resources/default_params.json");
+        assertTrue(PopulationParameters.get().isValid());
+    }
+
+    @Test (expected = AssertionError.class)
+    public void testIsInValid() throws IOException {
+        //Test that invalid parameters are handled correctly
+        ParameterReader.readParametersFromFile("src/test/resources/test_invalid_params.json");
+        assertTrue(PopulationParameters.get().isValid());
+    }
+
+    @Test (expected = InvalidParametersException.class)
+    public void testNullParameters() {
+        //Test that null parameters are handled correctly
+        assertTrue(PopulationParameters.get().isValid());
```

Comment:
I suggest removing the `assertTrue()` here too:
```suggestion
        PopulationParameters.get().isValid();
```

Reply:
ok